src/usr/lib/rsetup/tui/task/baota/baota.sh

```diff
@@ -1,31 +1,9 @@
 # shellcheck shell=bash
-
-__task_baota_uninstall() {
-    if yesno "Are you sure to uninstall Baota?"
-    then
-        if uninstall_baota
-        then
-            msgbox "Baota was successfully uninstalled."
-        else
-            msgbox "Failed to uninstall Baota." "$RTUI_PALETTE_ERROR"
-        fi
-    fi
-}
-
-__task_baota_install() {
-    if yesno "Are you sure you want to install Baota?"
-    then
-        install_baota
-    fi
-}
-
-__task_baota() {
-    menu_init
-    if [[ -f /www/server/panel/BT-Panel ]]
-    then
-        menu_add __task_baota_uninstall   "Uninstall Baota"
-    else
-        menu_add __task_baota_install     "Install Baota"
-    fi
-    menu_show "Please select an option below:"
-}
+if [[ -f /www/server/panel/BT-Panel ]]
+then
+    checklist_add "Uninstall Baota" "OFF"
```

Comment:
```suggestion
    checklist_add "Uninstall Baota" "OFF" “uninstall_baota”
```
又看了下有专门做一个RTUI_CHECKLIST_VALUE数组来保存自定义数据，然后用checklist_getitem来获取数据，应该不需要RSETUP_TASK_CALLBACKS
需要弄一个API参考了